src/OI.cpp

```diff
@@ -1,101 +1,92 @@
-// RobotBuilder Version: 2.0
-//
-// This file was generated by RobotBuilder. It contains sections of
-// code that are automatically generated and assigned by robotbuilder.
-// These sections will be updated in the future when you export to
-// C++ from RobotBuilder. Do not put any code or make any change in
-// the blocks indicating autogenerated code or it will be lost on an
-// update. Deleting the comments indicating the section will prevent
-// it from being updated in the future.
-
-
 #include "OI.h"
-
-// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
 #include "SmartDashboard/SmartDashboard.h"
 #include "Commands/AutonomousCommand.h"
 #include "Commands/MecanumDrive.h"
-#include "Commands/defaultElevator.h"
-#include "Commands/defaultIntake.h"
-#include "Commands/defaultLights.h"
-#include "Commands/ElevatorUp.h"
-#include "Commands/ElevatorDown.h"
-#include "Commands/HaySqueezeOpenClose.h"
-#include "Commands/IntakeIn.h"
-#include "Commands/IntakeOut.h"
-#include "Commands/IntakeOpenClose.h"
-
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=INCLUDES
+#include "Commands/ElevatorOn.h"
+#include "Commands/IntakeOn.h"
+#include "Commands/SetActuator.h"
+#include "Commands/SetBrake.h"
+#include "Commands/SetHaySqueeze.h"
+#include "Commands/ToggleActuator.h"
+#include "Commands/ToggleBrake.h"
+#include "Commands/ToggleHaySqueeze.h"
+
 
 OI::OI() {
     // Process operator interface input here.
-    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
-    coDriver.reset(new Joystick(1));
-    
-    coLGreen.reset(new JoystickButton(coDriver.get(), 15));
+
+ //TODO find out and replace all old button controls
+	coDriver.reset(new Joystick(1));
```

Comment:
Magic Number? Meaning if we make a constant for the button ids we might as well make one for the controllers